src/payment/strategies/cybersource/cybersource-payment-strategy.ts

```diff
@@ -0,0 +1,72 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import {
+    MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType,
+} from '../../../common/error/errors';
+import OrderRequestBody from '../../../order/order-request-body';
+import { CreditCardInstrument } from '../../payment';
+import PaymentMethod from '../../payment-method';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import CyberSourcePaymentProcessor from './cybersource-payment-processor';
+import CyberSourceThreeDSecurePaymentProcessor from './cybersource-threedsecure-payment-processor';
+
+export default class CyberSourcePaymentStrategy implements PaymentStrategy {
+    private _processor?: CyberSourcePaymentProcessor | CyberSourceThreeDSecurePaymentProcessor;
```

Comment:
@davidchin Right now we are using the same strategy to handle both flows, do you agree with this approach? Or, maybe we could just override paymentMethod from right on the backend and just have two distinct strategies with one single processor.